templates/careers/departments.html

```diff
@@ -0,0 +1,108 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1Kh0-nMGMfpsyKITGUw2jGMGyB3eNV9M9x_Ap0FowzpA/{% endblock meta_copydoc %}
+
+{% block title %}Careers{% endblock %}
+
+{% block meta_description %}Canonical publishes Ubuntu, provides commercial services and solutions for Ubuntu, and works with hardware manufacturers, software vendors and public clouds to certify Ubuntu.{% endblock %}
+
+{% block content %}
+<section class="p-strip--light is-deep">
+  <div class="row">
+    <div class="col-7">
+      <h1 class="p-heading--2">Departments at Canonical</h1>
+    </div>
+    <div class="col-5">
+      <p>Life at Canonical is anything but corporate. As a company that exists to support Ubuntu, one of today's most important open source projects, we are changing the world on a daily basis. It’s a collaborative environment, but one in which every member of the team takes personal responsibility for everything they produce.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-4">
+      <h2>Engineering</h2>
+      <p>Open source is transforming the entire stack. This is your chance to be right at the center of that revolution, to shape the platforms and tools that millions use to invent our global technology future. From bare metal to cloud to high performance computing, from AI and big data to the web and connected devices, open source is the key ingredient for success.</p>
+      <a href="/careers/all?filter=Engineering">Check out open Engineering roles</a>
```

Comment:
```suggestion
      <a href="/careers/all?filter=Engineering">Check out open Engineering roles&nbsp;&rsaquo;</a>
```
They all need the little arrow also

Reply:
I believe we're not using this convention on these pages, but just to double check @marinacastejon could you confirm pls? 🙏🏼

Reply:
Follow-up: Lyubo confirmed that we aren't using the chevrons 👍🏻